app/models/school_device_allocation.rb

```diff
@@ -19,4 +22,18 @@ def self.by_computacenter_device_type(cc_device_type)
   def computacenter_cap_type
     Computacenter::CapTypeConverter.to_computacenter_type(device_type)
   end
+
+private
+
+  def validate_cap_lte_allocation
+    if cap > allocation
+      errors.add(:cap, "can't be greater than allocation")
+    end
+  end
+
+  def validate_cap_gte_devices_ordered
+    if cap < devices_ordered
+      errors.add(:cap, "can't be less than devices ordered")
```

Comment:
can we use [a custom key](https://stackoverflow.com/a/18185133) to push this error message into `en.yml`?

Reply:
done